source/appModules/searchui.py

```diff
@@ -18,6 +18,15 @@ class StartMenuSearchField(SearchField):
 
 class AppModule(appModuleHandler.AppModule):
 
+	def event_NVDAObject_init(self, obj):
+		if isinstance(obj, UIA):
+			# #10341: Build 18363 introduces modern search experience in File Explorer.

```

Comment:
If this is only available in Build 18363 and later, should this be protected by a check for the windows version? If not, please describe in the comment how this code handles earlier versions. I assume that the `UIAAutomationId` wont be `suggestionCountForNarrator` in earlier builds of Windows?